flang/lib/Optimizer/CodeGen/Target.cpp

```diff
@@ -825,6 +825,48 @@ struct TargetAArch64 : public GenericTarget<TargetAArch64> {
     }
     return marshal;
   }
+
+  static bool isHFA(fir::RecordType ty) {
```

Comment:
What is HFA? Please document.

Reply:
@DavidTruby I think you missed this when responding to feedback

Reply:
Sorry I commented on the wrong one here. I meant the "expand auto" nit below.